source/about/faq.rst

```diff
@@ -12,22 +12,22 @@ For Everyone
 What is Sponge?
 ---------------
 
-Sponge is a new versatile Minecraft: Java Edition API. It was made to enrich your Minecraft experience by allowing
-plugins to add functionality to Minecraft. Read more about Sponge here: :doc:`introduction` and about the history of
+Sponge is a versatile Minecraft: Java Edition API. It was made to enrich your Minecraft experience by enabling creation 
+of plugins to add functionality to Minecraft. Read more about Sponge here: :doc:`introduction` and about the history of
 Sponge here: :doc:`history`
 
 What is required to run Sponge?
 -------------------------------
 
 Sponge (and Minecraft) needs the Java Runtime Environment to run properly. You will obviously need a computer to run
-the server on too, besides that nothing is required. Learn more about choosing the correct Java version here:
-:doc:`../server/getting-started/jre/`
+the server on too, besides that nothing is required. Learn more about choosing and installing the correct Java version
+here: :doc:`../server/getting-started/jre/`
 
 Where do I get Sponge?
 ----------------------
 
-Beta development builds of SpongeForge and SpongeVanilla are already available:
-`Sponge Downloads <https://spongepowered.org/downloads>`_
+You can download both stable and experimental builds of SpongeForge and SpongeVanilla on our 
```

Comment:
`You can download SpongeForge and SpongeVanilla on our..`

wanna get away from that stupid stable/experimental naming, don't add it to the docs pls

Reply:
Sure.